src/connections/Entity.ts

```diff
@@ -91,7 +91,7 @@ export type EntityIndexConnection<T extends EntityDtoType> = {
   included?: JsonApiResource[];
 };
 
-type EntityIndexFilterKey = keyof Omit<
+export type EntityIndexFilterKey = keyof Omit<
```

Comment:
As mentioned in the v3 PR, I'd like to avoid re-using the entity query params for the project pitches controller. It should get its own that is specific to pitches.